bin/exit/main.go

```diff
@@ -63,8 +63,14 @@ func main() {
 	}
 
 	if config.Reverse {
-		for serviceName := range config.Services {
-			e := tuna.NewTunaExit(config, services, wallet)
+		var services []tuna.Service
+		err = tuna.ReadJson(opts.ServicesFile, &services)
+		if err != nil {
+			log.Fatalln("Load service file error:", err)
+		}
+		for serviceName, _ := range config.Services {
+			entryToExitMaxPrice, exitToEntryMaxPrice, err := tuna.ParsePrice("0.001")
```

Comment:
For the reverse tuna, on the exit side, user should use `ReverseMaxPrice` as the max acceptable price; on the entry side, provider should use `ReversePrice` to specify actual charge price and include it in subscribe metadata. You can check how this works in the forward mode